.circleci/config.yml

```diff
@@ -1,7 +1,22 @@
 version: 2.1
 orbs:
   node: circleci/node@2.1.1
+  python: circleci/python@0.3.2
 jobs:
+  downloadPGEData:
+    docker:
+      - image: circleci/python:latest
+    environment:
+      - PGE_STOP_DATE=7/5/20
```

Comment:
This part must be manually edited... unless perhaps we allow for errors to occur and search until 12/31/20 but that's not ideal

Reply:
@peuanthinsan can you undo your changes to this file?